NEWS.rst

```diff
@@ -18,6 +18,7 @@
 - Fix a crash on non ASCII characters from `lsb_release`. (#4062)
 - Fix an SyntaxError in an unused module of a vendored dependency. (#4059)
 - Fix UNC paths on Windows. (#4064)
+- Improve extras dependency handling (#3878)
```

Comment:
We've had a change in process; so instead of modifying NEWS.rst, you'd want to add a file in `news/` directory. More about that here: https://pip.pypa.io/en/latest/development/#adding-a-news-entry